src/WalkingModule/app/robots/iCubGazeboV2_5/dcm_walking/joypad_control/robotControl.ini

```diff
@@ -8,7 +8,7 @@ joints_list             ("torso_pitch", "torso_roll", "torso_yaw",
 
 # if 1 the joint is in stiff mode if 0 the joint is in compliant mode
 joint_is_stiff_mode     (true, true, true,
-                         false, false, true, true,
+                         false, true, true, true,
                          true, true, true, true,
                          true, true, true, true, true, true,
```

Comment:
Is there a reason why the left and right arms are not controlled with the same control mode?

Reply:
Sorry I forgot to edit that. Now are all in the stiff mode.